lib/users/lms/award-badge.js

```diff
@@ -14,10 +15,14 @@ function awardLMSBadge (args, cb) {
   var user = certif.user;
 
   function checkTestStatus (waterfallCb) {
-    if (certif.header.webHookType !== 'course_completion')
-      return cb(null, 'Unhandled webhook');
-    if (certif.enrollmentStatus !== 'passed' && certif.enrollmentStatus !== 'completed' )
-      return cb(null, 'Unhandled status');
+    if (certif.header.webHookType !== 'course_completion') {
+      return cb(null, {ok: false, why: 'Unhandled webhook'});
+    }
+    if (!_.every(certif.modules, {'status': 'passed'}) &&
+      !_.every(certif.modules, {'status': 'completed'}) &&
+      !_.includes(['completed', 'passed'], certif.enrollmentStatus) ) {
```

Comment:
No, the ands make sense. modules having a status of passed or completed, and enrollmentStatus of completed or passed are scenarios we support. We're checking here if none of those scenarios are true, and if so throwing the `ok: false`. If you changed this to ors, we'd potentially always get `ok: false`. This was done because of our little French issue with enrollmentStatus.

My concern about this though, is what if enrollmentStatus isn't completed or passed, but there's a mix of passed and completed in the modules. We'd end up in an unhandled status scenario.